content/blog/2017-06-14-interview-with-latelier-animation.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Interview with L’Atelier Animation
+created_at: 2017-06-14
+kind: article
+author_name: Brian Brazil
+---
+
+*Continuing our series of interviews with users of Prometheus, Philippe Panaite
```

Comment:
+*Continuing our series of interviews with users of Prometheus, Philippe Panaite and Barthelemy Stevens